etc/webpack/webpack.config.js

```diff
@@ -12,13 +12,15 @@ const DIST_PATH = path.resolve(PROJECT_ROOT_PATH, 'dist');
 const PACKAGES_PATH = path.resolve(SOURCE_PATH, 'packages');
 const NODE_MODULES_PATH = path.resolve(SOURCE_PATH, '../node_modules');
 
+const LIBRARY_NAME = require('../../package.json').name;
```

Comment:
`const { name } = require('../../package.json');`?

Or

```
const package = require('../../package.json');
const { name } = package;
```

...matter of taste